src/main/java/by/antonsh/arrays2/Ex1.java

```diff
@@ -0,0 +1,35 @@
+package by.antonsh.arrays2;
+
+import java.util.Arrays;
+import java.util.Random;
+
+/*
+ Найти произведение элементов, кратных 3.
+ */
+public class Ex1 {
+    public static void main(String[] args) {
+        Random random = new Random();
+        int[] arrayInts;
+        arrayInts = random.ints(6, 0, 10).toArray();
+        int result=1;
+
+        System.out.println(Arrays.toString(arrayInts));
+        for (int i = 0; i < arrayInts.length; i++) {
+            if (arrayInts[i] % 3 == 0 && arrayInts[i]!=0) {
```

Comment:
it is better first to check arrayInts[i] != 0, it is false we don't need to divide

Reply:
but will change it)